scripts/install

```diff
@@ -179,7 +179,7 @@ ynh_store_file_checksum "${final_path}/config/config.inc.php"
 #=================================================
 
 # Set permissions to app files
-chown -R root: "$final_path"
+chown -R $app: "$final_path"
```

Comment:
You should limit your owner change to the strict minimum, so to the enigma folder or plugins folder.
Furthermore, you should also handle the upgrade script as well 😉